plugins/govppmux/plugin_impl_govppmux.go

```diff
@@ -181,7 +188,12 @@ func (plugin *GOVPPPlugin) NewAPIChannel() (govppapi.Channel, error) {
 	if plugin.replyTimeout > 0 {
 		ch.SetReplyTimeout(plugin.replyTimeout)
 	}
-	return ch, nil
+	retryCfg := retryConfig{}
+	if plugin.config != nil {
+		retryCfg.attempts = plugin.config.RetryRequestCount
```

Comment:
Perhaps we should just unify the way we deal with config here. Just above this we use `replyTimeout` from the plugin's field, set in `Init`, we could just use `Config` directly and get rid of the fields.